product_docs/docs/edbcloud/beta/getting_started/01_check_resource_limits.mdx

```diff
@@ -8,6 +8,8 @@ The default number of total cores per subscription per region is 20. See [Virtua
 
 The default Public IP address limits for Public IP Addresses Basic and Public IP Addresses Standards is set to 10. See [Public IP address limits](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/azure-subscription-service-limits#publicip-address) for more information. You need to increase the limit of `Public IP Addresses - Basic` and `Public IP Addresses - Standard` for the regions where you plan to deploy your EDB clusters with the total number of EDB clusters you plan to use.
 
+<iframe width="560" height="315" src="https://www.youtube.com/embed/cfl11ajrPhE" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
Should use the `title` attribute to describe the video a bit more - maybe "Video walkthrough of raising Azure resource limits" or the like. Can be long if need-be.